docs/backend/setup/database-migrations.md

```diff
@@ -7,10 +7,11 @@ sidebar_position: 4
 If you need to change the database schema, you'll need to write a migration. These are the general steps you'll need to follow:
 
 1. [Generate a new migration file](#Creating-Migrations)
-1. [Add the new SQL to the generated file](#writing-migrations)
-1. [Set up your database](#Setup)
-1. [Run the migrations](#Running-Migrations)
-1. Test your new migration
+2. [Add the new SQL to the generated file](#writing-migrations)
+3. [Set up your database](#Setup)
+4. [Run the migrations](#Running-Migrations)
+5. [Delete a migration](#Delete-Migrations)
+6. Test your new migration
```

Comment:
I believe that docusaurus automatically adds the right numbers when it renders the real page so these should all be `1.` still.

Reply:
Ok, thanks!